qual_id/pattern.py

```diff
@@ -46,6 +48,7 @@ class Pattern:
         "instrument": Instrument(),
         "language": Language(),
         "music": Music(),
+        "operatingsystem": OperatingSystem(),
         "planet": Planet(),
```

Comment:
Are you forgetting to add "programminglanguage" to the map?

Reply:
oops 😉